sdk/python/ragflow/__init__.py

```diff
@@ -1,3 +1,5 @@
 import importlib.metadata
 
 __version__ = importlib.metadata.version("ragflow")
+
+from .ragflow import RAGFlow
```

Comment:
???

Reply:
It means that I import this class from the same level of directory.